OAuth/ResourceOwner/FiwareResourceOwner.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ * This file is part of the HWIOAuthBundle package.
+ *
+ * (c) Hardware.Info <opensource@hardware.info>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace HWI\Bundle\OAuthBundle\OAuth\ResourceOwner;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+use Buzz\Message\RequestInterface as HttpRequestInterface;
+
+/**
+ * FiwareResourceOwner
```

Comment:
this should be removed IMO. the short description of the class should be the line ``Resource owner for the fiware keyrock idm oauth 2.0 service``. Putting the class name as short description in the phpdoc brings exactly 0 value